components/ModalRoot.vue

```diff
@@ -22,6 +29,8 @@ export default {
   setup() {
     const componentRef = ref(null)
     const titleRef = ref("")
+    const isSmallModal = ref(false)
```

Comment:
Should we just make the modal responsive? instead of requiring an extra prop to add the extra class

Reply:
Actually it's using sfModal and by default which takes full screen in mobile view which is the default behavior for all mockups like login, addTocartConformation etc but for this scenario modal is not taking entire screen in mobile that's why I made it conditional. So, should I keep it like that or take diff approach.